File: beginner/pagina01/1019/1019.ts
const fs = require("fs")
const input = fs.readFileSync('/dev/stdin', 'utf-8')
const num = Number(input)

function TimeConvert(n) {

    let hours = Math.floor(n / 3600);
    let minu = Math.floor((n - (hours * 3600)) / 60);
    let second = Math.floor(n % 60);

    console.log(`${hours.toFixed(0)}:${minu.toFixed(0)}:${second}`);

}

TimeConvert(num);

File: beginner/pagina01/1019/1019.rs
use std::io;

fn main() {
let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();

    let value: u32 = input.trim().parse().unwrap();

    let hours = value / 3600;
    let minutes = (value - (hours * 3600)) / 60;
	let seconds = value % 60;

    println!("{}:{}:{}", hours, minutes, seconds);
}